packages/cli/src/commands/server/watchMode.js

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Since we are migrating the whole codebase to Typescript, can we also have this PR written in it?

Reply:
I will start the work on this.